neuromation/api/config.py

```diff
@@ -266,18 +272,28 @@ def _save(cls, config: _Config, path: Path) -> None:
             payload["version"] = config.version
             payload["cluster_name"] = config.cluster_name
         except (AttributeError, KeyError, TypeError, ValueError):
-            raise ConfigError("Malformed config. Please logout and login again.")
-
-        # atomically rewrite the config file
-        # forbid access to other users
-        def opener(file: str, flags: int) -> int:
-            return os.open(file, flags, 0o600)
+            raise ConfigError(MALFORMED_CONFIG_MSG)
 
         path.mkdir(0o700, parents=True, exist_ok=True)
-        with atomicwrites.atomic_write(
-            str(path / "db"), opener=opener, overwrite=True
-        ) as f:
-            yaml.safe_dump(payload, f, default_flow_style=False)
+
+        config_file = path / "db"
+        with sqlite3.connect(str(config_file)) as db:
+            # forbid access to other users
+            os.chmod(config_file, 0o600)
+
+            db.row_factory = sqlite3.Row
+            _init_db_maybe(db)
+
+            cur = db.cursor()
+            content = yaml.safe_dump(payload, default_flow_style=False)
+            cur.execute("DELETE FROM main")
+            cur.execute(
+                """
+                INSERT INTO main (content, timestamp)
+                VALUES (?, ?)""",
+                (content, time.time()),
```

Comment:
`time.time()` is float, but the timestamp column is INTEGER. I suppose this will cause a warning in 3.8 and error in 3.10.

Where the timestamp is used?

Reply:
Nowhere. I wanted to save the last update time. 
The second-round precision is good enough, but I can change the type to REAL